test/backbone.model.js

```diff
@@ -63,12 +63,26 @@ describe('Backbone.Model', function () {
 
         it('removes from localForage', function(done) {
             model.destroy({
-                success: function() {
-                    model.fetch({
-                      error: function () {
-                        done();
+                success: function(model, resp, options) { // jshint unused:false
+                    expect(model.attributes).toEqual(resp);
+                    var handlers = {
+                      success: function() {
+                        testComplete();
+                      },
+                      error: function() {
+                        testComplete();
```

Comment:
@tofumatt there was no configuration enforcing a 4 spaces indentation. I've sent you a PR in localForage as well.